runtime/doc/options.txt

```diff
@@ -3267,7 +3267,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 				     >:SignColumn,B:SpellBad,P:SpellCap,
 				     R:SpellRare,L:SpellLocal,-:Conceal,
 				     +:Pmenu,=:PmenuSel,x:PmenuSbar,
-				     X:PmenuThumb")
+				     X:PmenuThumb,q:QuickFixLine")
```

Comment:
> but maybe the expect() error output should remain a little bit cumbersome just to have an excuse for mentioning snapshot_util() in the error message

Hah, fair enough.  Docs would definitely help.  I've gotten by by tag jumping and reading what I can that way, but often miss the big block of text at the top of the file.

And yeah, @mhinz pointed out `snapshot_util()` to me and I got confused when I found no other use of it.

The takeaway from all this is that I'm removing the `'highlight'` changes later tonight 😜

As a side note, this has been useful for editing the screen tests: `match ErrorMsg /\[\[\_.\{-}\zs\^/`